enginetest/queries/query_plans.go

```diff
@@ -776,20 +773,20 @@ var PlanTests = []QueryPlanTest{
 	},
 	{
 		Query: `select * from ab where exists (select * from uv where a = 1)`,
-		ExpectedPlan: "Project\n" +
-			" ├─ columns: [ab.a:0!null, ab.b:1]\n" +
-			" └─ HashJoin\n" +
-			"     ├─ Eq\n" +
-			"     │   ├─ ab.a:0!null\n" +
-			"     │   └─ 1 (tinyint)\n" +
-			"     ├─ Table\n" +
-			"     │   ├─ name: ab\n" +
-			"     │   └─ columns: [a b]\n" +
-			"     └─ HashLookup\n" +
-			"         ├─ source: TUPLE(ab.a:0!null)\n" +
-			"         ├─ target: TUPLE(1 (tinyint))\n" +
-			"         └─ CachedResults\n" +
-			"             └─ Distinct\n" +
+		ExpectedPlan: "HashJoin\n" +
+			" ├─ Eq\n" +
+			" │   ├─ ab.a:0!null\n" +
+			" │   └─ 1 (tinyint)\n" +
+			" ├─ Table\n" +
+			" │   ├─ name: ab\n" +
+			" │   └─ columns: [a b]\n" +
+			" └─ HashLookup\n" +
+			"     ├─ source: TUPLE(ab.a:0!null)\n" +
+			"     ├─ target: TUPLE(1 (tinyint))\n" +
+			"     └─ CachedResults\n" +
+			"         └─ Distinct\n" +
+			"             └─ Project\n" +
+			"                 ├─ columns: []\n" +
```

Comment:
this is correct but interesting. will have to optimize this further at some point, collapse the join into an indexed table scan.

Reply:
Should I open an issue against GMS for this?

Reply:
this is another "nice to have", which I am not sure how to organize well. you could add a comment, issue, not super opinionated.